### DISCUSSION TITLE
range wifi stream

### DISCUSSION THREAD

**Discussion by dhamidou** [bitcraze/discussions#1322] [Q&A] [answered]
Hi, I'm currently using the Aideck and I've noticed that from a certain distance (about 3-4 m), the video stream is no longer continuous. Can you tell me if this is normal?
Thanks in advance

**Comment by gemenerik** [comment]
With non-continuous do you mean it is dropping frames or that the stream stops completely? Is the deck serving a hotspot or are you connecting it to an external network?

**Reply by dhamidou** [reply, marked as answer]
Hi, I mean it is dropping frames and the deck serving a hotspot. But I solved my problem, it seems that the frequency of my topic which is used to publish images from the camera was too low.
